gadget/install/install.go

```diff
@@ -232,6 +232,15 @@ func createPartitions(model gadget.Model, gadgetRoot, kernelRoot, bootDevice str
 
 	// auto-detect device if no device is forced
 	// device forcing is used for (spread) testing only
+	if bootDevice == "" {
+		foundDisk, err := disks.DiskFromMountPoint("/run/mnt/ubuntu-seed", nil)
+		if err == nil {
+			bootDevice = foundDisk.KernelDeviceNode()
+		} else {
+			logger.Noticef("Warning: cannot find disk from mounted seed: %s", err)
+			bootDevice = ""
+		}
+	}
 	if bootDevice == "" {
```

Comment:
Maybe this part can be moved inside the error case above (`foundDisk.KernelDeviceNode()` should always return something if there was no error I guess?)